### ISSUE TITLE
Audio and music tracks are to loud

### ISSUE THREAD

**Issue by SimonStnn** [Comon-tech/numQuest#33]
Make a volume slider in the settings sidebar.

**Comment by C-o-m-o-n** [comment]
> Make a volume slider in the settings sidebar.

is it working now, when deployed?

**Comment by SimonStnn** [comment]
I'm not sure what you mean. No new changes were pushed to the `main` branch.

**Comment by SimonStnn** [comment]
The volume can be changed with the `volume` property:

```js
const audio = new Audio("");
audio.volume = 0.2;
```

It is a value between 0.0 and 1.0:

- 1.0 is the hightest volume (default)
- 0.0 is silent (mute)

I suggest this slider gets added, and when playing a sound the value of this slider gets parsed to a number and devided by 100 when setting the volume.

```html
<input type="range" id="volume-slider" min="0" step="1" max="100">
```

**Comment by LyubomirT** [comment]
@C-o-m-o-n May I take on this issue as well? Sounds simple enough to me.

**Comment by C-o-m-o-n** [comment]
> @C-o-m-o-n May I take on this issue as well? Sounds simple enough to me.

That would be very  much appreciated

**Comment by LyubomirT** [comment]
Fixed in #45.
